dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/sfdr/SfdrFramework.kt

```diff
@@ -0,0 +1,52 @@
+package org.dataland.frameworktoolbox.frameworks.sfdr
+
+import org.dataland.frameworktoolbox.frameworks.InDevelopmentPavedRoadFramework
+import org.dataland.frameworktoolbox.intermediate.Framework
+import org.dataland.frameworktoolbox.intermediate.group.ComponentGroup
+import org.dataland.frameworktoolbox.intermediate.group.ComponentGroupApi
+import org.dataland.frameworktoolbox.intermediate.group.edit
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.LabelBadgeColor
+import org.dataland.frameworktoolbox.template.components.ComponentGenerationUtils
+import org.springframework.stereotype.Component
+import java.io.File
+
+/**
+ * Definition of the SFDR framework
+ */
+@Component
+class SfdrFramework : InDevelopmentPavedRoadFramework(
+    identifier = "sfdr",
+    label = "SFDR",
+    explanation = "Sustainability Finance Disclosure Regulation",
+    File("./dataland-framework-toolbox/data-dictionary/csv/dataDictionary-SFDR (developer version).csv"),
```

Comment:
Den Namen sollten wir anpassen.

Reply:
Fixed by splitting off sfdr into own xlsx file and implementing xlsx parsing directly into the framework toolbox.